Tests/MapboxMapsTests/Gestures/GestureHandlers/PinchBehaviors/PinchBehaviorProviderTests.swift

```diff
@@ -12,7 +12,8 @@ final class PinchBehaviorProviderTests: XCTestCase {
                 rotateEnabled: true,
                 initialCameraState: .random(),
                 initialPinchMidpoint: .random(),
-                initialPinchAngle: .random(in: 0..<2 * .pi))
+                initialPinchAngle: .random(in: 0..<2 * .pi),
+                focalPoint: .random())
```

Comment:
```suggestion
                focalPoint: .random(.random()))
```

This will give you nils sometimes too. Probably not super important for this test since we expect it to just be a pass-through, but I wanted to make sure you were aware of it.

Reply:
Thanks, I wasn't aware of this indeed!